packages/graphql-hooks/package.json

```diff
@@ -48,7 +48,6 @@
     "react": "^17.0.0",
     "react-dom": "^17.0.0",
     "react-test-renderer": "^17.0.0",
-    "rollup": "^2.0.0",
```

Comment:
![Screenshot 2021-11-18 at 12 12 30](https://user-images.githubusercontent.com/630796/142413813-339f08ce-21dd-4fe9-b40d-2d996a8a2738.png)

This is mentioned among other Lerna warnings, but it needed to be addressed.
It was somehow causing this documented issue: https://reactjs.org/warnings/invalid-hook-call-warning.html
Probably by generating some mismatching version in the bundle.